src/Continuous/exponentiation.jl

```diff
@@ -120,94 +221,124 @@ A matrix.
 If ``A`` is invertible, ``Φ₁`` can be computed as
 
 ```math
-Φ₁(A, δ) = A^{-1}(e^{δA} - I_n),
+Φ₁(A, δ) = A^{-1}(e^{Aδ} - I_n),
 ```
 where ``I_n`` is the identity matrix of order ``n``.
 
 In the general case, implemented in this function, it can be
 computed as submatrices of the block matrix
 
 ```math
-P = \\exp \\begin{pmatrix}
-Aδ && δI_n && 0 \\\\
-0 && 0     && δI_n \\\\
-0 && 0     && 0
+P_{2n} = \\exp \\begin{pmatrix}
```

Comment:
As in the SpaceEx paper, this was the same 3n x 3n matrix used to compute the series Φ2 for reachability in dense time. However, it was found by @guberger ([here](https://github.com/dionysos-dev/Dionysos.jl/issues/66#issuecomment-761018455)) that for Φ1 it is enough to compute the principal 2n x 2n block of P.

Reply:
In some cases, we can restrict to a `(n+1) x (n+1)` block, I think: https://github.com/dionysos-dev/Dionysos.jl/issues/66#issuecomment-761018455

Reply:
true -- i would recommend doing that trick also in the more general case that the input set is known to be constant, ie. `x' = Ax + Bu` with `u(t) \in U` with `u' = 0` . there is a function` homogeneize` in this package exactly for that kind of transformation. also interesting (but it you have to do it manually) in the more general case that the dynamic equation of the input are known as in [this example](https://juliareach.github.io/ReachabilityAnalysis.jl/dev/man/pde/#Nonlinear-loads-1).